### DISCUSSION TITLE
How to we Manage LTR to RTL Conversion of the Site.

### DISCUSSION THREAD

**Discussion by SalmanShahid888** [reactjs/ur.react.dev#8] [Q&A] [answered]
@yousufkalim How do we manage RTL, Should We convert it to RTL or we should only focus on Translation right now?

**Comment by yousufkalim** [comment, marked as answer]
@SalmanShahid888 For now, let's just focus on translation, we'll convert it to RTL after 70% of completion.

Thank u so much for your interest and contribution.
